build.gradle

```diff
@@ -143,15 +143,19 @@ dependencies {
         }
     }
 
+
+    apt "com.jakewharton:butterknife-compiler:8.0.1"
+
     compile "com.android.support:appcompat-v7:23.4.0"
     compile "com.android.support:design:23.4.0"
     compile "com.android.support:support-annotations:23.4.0"
     compile "com.android.support:support-v4:23.4.0"
+    compile 'com.github.bmelnychuk:atv:1.2.8'
+    compile 'com.github.johnkil.print:print:1.2.2'
     compile "com.github.lecho:hellocharts-library:1.5.6@aar"
     compile "com.google.android.gms:play-services:9.0.1"
     compile "com.google.zxing:android-integration:3.2.1"
     compile "com.jakewharton:butterknife:8.0.1"
-    apt "com.jakewharton:butterknife-compiler:8.0.1"
     compile "com.jakewharton.timber:timber:3.1.0"
     compile "info.metadude.android:typed-preferences:2.0.0"
     compile "org.jboss.aerogear:aerogear-android-authz:3.0.0"
```

Comment:
@pilhuhn Aerogear lib have been 3.0.0 as max and have been placed to that in last PR.

Working with the recent accepted PR from MavenLocal